src/libraries/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -460,6 +460,20 @@ public void TestHasExited()
             }
         }
 
+        [Fact]        
+        public void TestToString_OnExitedProcess()
+        {
+            var p = Process.Start("dotnet");
```

Comment:
I am not sure we can be certain that "dotnet" is on the path. I would follow the pattern of other tests, which is to use ` Process process = CreateProcessLong();`. (This just makes a sleeping process) something like
```c#
var p = CreateProcessLong();
var name = p.ProcessName;
Assert.Equal($"System.Diagnostics.Process ({name})", p.ToString());
p.Kill();
Assert.Equal("System.Diagnostics.Process", p.ToString());
```
?

Reply:
otherwise looks good to me. thanks!